### ISSUE TITLE
Specify name of pre-rendered feature info property in init file, don't default to "description"

### ISSUE THREAD

**Issue by stevage** [TerriaJS/terriajs#1452]
From https://github.com/TerriaJS/neii-viewer/issues/21

Take a GeoJSON feature with `name` and `description` properties attributes:

```
            "properties": {
                "name": "Site 1",
                "description": "Site 1 description",
                "siteURL": "http://siteURL",
                "srsName": "epsg:4979",
                "operatingAuthority": {
                    "name": "Operating Authority URL",
                    "url": "http://operatingAuthority"
                },
                "siteStatus": "active",
```

These attributes get shown without the names of the attributes themselves (perhaps desirable), but no other attributes show up:

![namedesc](https://cloud.githubusercontent.com/assets/678344/14626468/ebe4ffc2-062e-11e6-8125-dedcc5956348.png)

Renaming them to something else fixes the problem:

![fixed](https://cloud.githubusercontent.com/assets/678344/14626477/0e3ec8dc-062f-11e6-8baf-991eed9483dc.png)

**Comment by stevage** [comment]
Hmm, it's in Cesium. [Here](https://github.com/TerriaJS/cesium/blob/master/Source/DataSources/GeoJsonDataSource.js#L192). 

Looks like we can maybe pass a `describe` option to `GeoJsonDataSource.load` which overrides that behaviour.

**Comment by kring** [comment]
My first thought was "well that's a nice feature that you can specify a `description` and the content gets rendered as markdown/html."  But maybe `description` is too common a property name to use in that way?

**Comment by stevage** [comment]
Yeah, it would be fine as an attribute of the GeoJsonDataSource itself, but as a content element - well, that's making a very big (and rather restrictive) assumption about the kinds of GeoJSON objects that are going to be shown on a Cesium map.

**Comment by kring** [comment]
I feel that it would be ok if, instead of `description`, it looked for `htmlDescriptionOfFeature` or something like that.  Do you agree?  It's nice for the feature to be able to provide totally custom HTML sometimes.

**Comment by stevage** [comment]
Hmm, yeah, but it's that murky world of trying to embed presentation metadata within data.

That'd probably be ok if it also had the "show raw data" mode.

**Comment by kring** [comment]
Sure but that murky world is pretty useful sometimes.  That's why we have things like simplestyles.  Having a "raw data" button too sounds good to me, though.

Perhaps the catalog item could specify the name of the property from which to pull finished markdown / html, and it defaults to `htmlDescriptionOfFeature`.  If the property doesn't exist you just get normal behavior (table of properties), so you can effectively force that by setting the catalog item property to something random.

**Comment by stevage** [comment]
Yeah, that sounds good. Maybe just `htmlDescription` as the default?

**Comment by kring** [comment]
Works for me.

**Comment by kring** [comment]
#2826 covers the suggestion that we add the "Show Raw Data" button this scenario.  This issue is to default to a property other than `description` and to allow it to be specified in the init file.
